src/yogita/BankData.java

```diff
@@ -0,0 +1,176 @@
+package yogita;
+
+/*
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI
+
+*/
+
+public class BankData {
+
+	float sbiBal, iciciBal, hdfcBal, bobBal;
+	int sbiCreditCnt, iciciCreditCnt, hdfcCreditCnt, bobCreditCnt;
+	int sbiDebitCnt, iciciDebitCnt, hdfcDebitCnt, bobDebitCnt;
+
+	static int totalCredit, totalDebit;
+
+	public void creditOperation(String bankName, int amount) {
+		switch (bankName) {
+		case "sbi":
+			sbiBal = sbiBal + amount;
+			sbiCreditCnt++;
+			totalCredit++;
+			System.out.println("Credit balance of SBI:" + sbiBal);
```

Comment:
No need SOP here,
we have one separate requirement to display credits n debit operation count.